.github/workflows/nightly.yml

```diff
@@ -19,21 +19,11 @@ on:
 jobs:
   brew-build:
     name: Build Semgrep via Brew from `returntocorp/semgrep:develop`
-    runs-on: ["self-hosted", "macOS", "X64"]
+    runs-on: macos-12
```

Comment:
those one I'm fine running with macos-12 instead of self-hosted. They can be slow. it's ok, it's run during the night.